### ISSUE TITLE
OneNote / Yahoo Mail / Gmail recipes : dark mode not working

### ISSUE THREAD

**Issue by Bunnnyface** [ferdium/ferdium-app#1154]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that Ferdium Recipes adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22recipe+%3Abookmark_tabs%3A%22+label%3A%22bug+%3Abeetle%3A%22+) for a bug report that matches the one I want to file, without success.

### Ferdium Version

6.2.6 portable

### Recipe Name

OneNote

### Recipe Version

1.3.0

### Steps to reproduce

1. Install recipe
2. Apply dark mode during installation
3. Apply/Unapply dark mode (Ctrl+Shift+D)

### Expected Behavior

Dark mode for OneNote should be activated since "dark mode" option is checked in the recipe and with Ctrl+Shift+D.
Every other recipes have their dark mode working.

### Actual Behavior

The screen to login is in dark mode but when connected, OneNote stays in light mode.

**Comment by Bunnnyface** [comment]
Hello everyone !

Does anybody has the same issue with onenote not displaying with darkmode ?
I can provide screenshots or other files to find the culprit if needed.

Have great time everybody !

**Comment by Bunnnyface** [comment]
Hi everyone !

As the previous comments on this thread said, dark mode doesn't work on OneNote.
But it does'nt work either on yahoo mail and Gmail.

Maybe these issues are related ?
Thanks by advance for your help !

**Comment by yarons** [comment]
Yes, I also tried adding the following to the darkmode.css file and it didn't help:
https://github.com/MichaelTontchev/onenote-dark-theme/blob/master/style.css
Maybe this file requires additional modifications.
